catalog/dags/data_refresh/create_filtered_index_dag.py

```diff
@@ -193,64 +141,25 @@ def point_alias(destination_index_suffix: str) -> TaskGroup:
         },
         render_template_as_native_obj=True,
     ) as dag:
-        prevent_concurrency = prevent_concurrency_with_data_refresh(
-            force="{{ params.force }}",
-        )
-
-        # If a destination index suffix isn't provided, we need to generate
-        # one so that we know where to point the alias
-        destination_index_suffix = generate_index_suffix(
-            "{{ params.destination_index_suffix }}"
-        )
-
-        get_current_index_if_exists = ingestion_server.get_current_index(target_alias)
-
-        do_create, await_create = create_and_populate_filtered_index(
-            origin_index_suffix="{{ params.origin_index_suffix }}",
-            destination_index_suffix=destination_index_suffix,
-        )
-
-        # Await healthy results from the newly created elasticsearch index.
-        index_readiness_check = ingestion_server.index_readiness_check(
-            media_type=media_type,
-            index_suffix=f"{destination_index_suffix}-filtered",
-        )
-
-        do_point_alias = point_alias(destination_index_suffix=destination_index_suffix)
-
-        delete_old_index = ingestion_server.trigger_task(
-            action="DELETE_INDEX",
-            model=data_refresh.media_type,
-            data={
-                "index_suffix": XCOM_PULL_TEMPLATE.format(
-                    get_current_index_if_exists.task_id, "return_value"
-                ),
-            },
+        prevent_concurrency = prevent_concurrency_with_data_refresh.override(
+            retries=0
+        )()
```

Comment:
I _might_ be wrong, but I think if this isn't called with any arguments, then it doesn't need to be called at all? The bitflow definition below should ensure it's actually run in the course of the DAG (_I think_).
```suggestion
        prevent_concurrency = prevent_concurrency_with_data_refresh.override(
            retries=0
        )
```

Reply:
This caused errors for me at DAG parsing in testing.